lib/pharos/config_schema.rb

```diff
@@ -153,6 +153,9 @@ def self.messages
         optional(:kubelet).schema do
           optional(:read_only_port).filled(:bool?)
         end
+        optional(:kubeadm).schema do
```

Comment:
IMHO this is not the right keyword for this... let's come up with something better:
- `api.use_proxy`
- `control_plane.use_proxy`
- `something_else.use_proxy`

@kke @jnummelin wdyt?

Reply:
I agree, we should not "leak" kubeadm to the config, there might be time where do not use it at all.

From the above suggestions, the best is `control_plane.use_proxy`.

Reply:
Alright, let's use `control_plane.use_proxy`.

Reply:
True that.  Control plane sounds fitting.